content/patterns/checkbox/examples/checkbox-mixed.html

```diff
@@ -18,103 +18,104 @@
-            <br>Note: The SVG element needs to have the CSS <code>forced-color-adjust</code> property set to <code>auto</code> for the <code>currentcolor</code> value to be updated in high contrast mode.
-            Some browsers do not use <code>auto</code> for the default value.
+            ハイコントラスト設定で色が反転した場合に、CSS のインライン SVG チェックボックスグラフィックの境界線が背景と十分なコントラストを持つようにするために、境界線の色はテキストコンテンツの色と同期されます。
+            たとえば、チェックボックスの境界線の色は、チェックボックスを描画するために使用される <code>rect</code> 及び <code>polyline</code> 要素の <code>stroke</code> プロパティに CSS <code>currentcolor</code> 値を指定することにより、ハイコントラストモードテキストの前景色と一致するように設定されます。
+            チェックボックスグラフィックの背景をハイコントラストの背景色と一致させるには、<code>rect</code> 要素の <code>fill-opacity</code> 属性をゼロに設定します。
+            代わりに特定の色を使用して <code>stroke</code> 及び <code>fill</code> プロパティを指定した場合、それらの色はハイコントラストモードで同じままであり、チェックボックスと背景の間のコントラストが不十分になったり、色がハイコントラストモードの背景と一致した場合はチェックボックスが見えなくなったりする可能性があります。
+            <br>注：SVG 要素は、ハイコントラストモードで <code>currentcolor</code> 値を更新するために、CSS <code>forced-color-adjust</code> プロパティを <code>auto</code> に設定する必要があります。
+            一部のブラウザーでは、デフォルト値に <code>auto</code> を使用しません。
           </li>
         </ul>
       </section>
 
       <section>
-        <h2 id="kbd_label">Keyboard Support</h2>
+        <h2 id="kbd_label">キーボードのサポート</h2>
         <table aria-labelledby="kbd_label" class="def">
           <thead>
             <tr>
-              <th>Key</th>
-              <th>Function</th>
+              <th>キー</th>
+              <th>機能</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="key-tab">
               <th><kbd>Tab</kbd></th>
-              <td>Moves keyboard focus among the checkboxes.</td>
+              <td>チェックボックス間でキーボードフォーカスを移動します。</td>
             </tr>
             <tr data-test-id="key-space">
               <th><kbd>Space</kbd></th>
               <td>
                 <ul>
-                  <li>Cycles the tri-state checkbox among unchecked, mixed, and checked states.</li>
-                  <li>When the tri-state checkbox is unchecked, all the controlled checkboxes are unchecked.</li>
-                  <li>When the tri-state checkbox is mixed, the controlled checkboxes return to the last combination of states they had when the tri-state checkbox was last mixed or to the default combination of states they had when the page loaded.</li>
-                  <li>When the tri-state checkbox is checked, all the controlled checkboxes are checked.</li>
+                  <li>三値状態のチェックボックスを、チェックなし、混合、チェック済みの状態で切り替えます。</li>
+                  <li>三値状態のチェックボックスがオフの場合、制御されているすべてのチェックボックスはオフになります。</li>
+                  <li>三値状態のチェックボックスが混合されている場合、制御されているチェックボックスは、三値状態のチェックボックスが最後に混合されたときに持っていた状態の最後の組み合わせ、又はページが読み込まれたときに持っていた状態のデフォルトの組み合わせに戻ります。</li>
+                  <li>三値状態のチェックボックスがオンの場合、制御されているすべてのチェックボックスがオンになります。</li>
```

Comment:
「〜の場合」がSpaceキーを押したあとの話をしているはずなのが、読みとりづらい。また他の場所ではmixedについては他の場所で「混合状態」と呼んでいるようなので、それに揃えたい。
「オン」「オフ（チェックなし）」については表記が揺れていそう。正確に状態を表現するのなら「チェック」という言葉を入れたほうがいいように思う。
「チェック済み」「未チェック」「チェックされている」「チェックされていない」あたりの言い方がされている模様。「チェックなし」はここでしか使われていない。

いったん「チェックされている状態」「チェックされていない状態」にしてみる。

```suggestion
                  <li>三値状態のチェックボックスを、チェックされていない状態、混合状態、チェックされているの状態で切り替えます。</li>
                  <li>三値状態のチェックボックスがチェックされていない状態に切り替わる場合、制御されているすべてのチェックボックスがチェックされていない状態になります。</li>
                  <li>三値状態のチェックボックスが混合状態に切り替わる場合、制御されているチェックボックスは、三値状態のチェックボックスが最後に混合状態となっていたときに持っていた状態の最後の組み合わせ、又はページが読み込まれたときに持っていた状態のデフォルトの組み合わせに戻ります。</li>
                  <li>三値状態のチェックボックスがチェックされている状態に切り替わる場合、制御されているすべてのチェックボックスがチェックされている状態になります。</li>
```

Reply:
ありがとうございます。

> 「〜の場合」がSpaceキーを押したあとの話をしているはずなのが、読みとりづらい。

原文からそのきらいはありますね……
「切り替わる」だとやや原文から離れてしまう感じがします。「となった」とするのはどうでしょうか？

---

<s>表記ゆれは統一したいですね。気づかずすみません。統一します。

この箇所について、最初の文は状態の名前を列挙していると考えると、
> Cycles the tri-state checkbox among unchecked, mixed, and checked states.

名詞的に「未チェック」「混合」「チェック済み」として、後続文でも同じ表現をする方がしっくりくるように感じます。表現も端的です。

> また他の場所ではmixedについては他の場所で「混合状態」と呼んでいるようなので、それに揃えたい。

他の箇所で「混合状態」としている箇所は "mixed-state" や "mixed state" となっていて、 "state" を訳出している形です。
ここでは状態の名前として、「混合」とするのがよいと考えます。

---

2つをまとめるとこんな感じでしょうか。いかがでしょうか……？

```suggestion
                <ul>
                  <li>三値状態のチェックボックスを、未チェック、混合、チェック済みの状態で切り替えます。</li>
                  <li>三値状態のチェックボックスが未チェックとなった場合、制御されているすべてのチェックボックスが未チェックになります。</li>
                  <li>三値状態のチェックボックスが混合となった場合、制御されているチェックボックスは、三値状態のチェックボックスが最後に混合されたときに持っていた状態の最後の組み合わせ、又はページが読み込まれたときに持っていた状態のデフォルトの組み合わせに戻ります。</li>
                  <li>三値状態のチェックボックスがチェック済みとなった場合、制御されているすべてのチェックボックスがチェック済みになります。</li>
```

---

全体の統一案として、以下はどうでしょうか？

- 状態の名前(名詞): 「未チェック、混合、チェック済み」
- 現在の状態(形容詞): 「チェックボックスがチェック済み」「チェックボックスが未チェック」
- 操作(動詞): 「チェックする」「チェックを解除する」

「オン」「オフ」に関しては、原文で "turned on or off" というような表現の箇所にだけ採用する</s>

Reply:
[MDN](https://developer.mozilla.org/ja/docs/Web/Accessibility/ARIA/Reference/Roles/checkbox_role#:~:text=%E3%80%8C%E3%83%81%E3%82%A7%E3%83%83%E3%82%AF%E3%81%95%E3%82%8C%E3%81%9F%E3%80%8D%E3%81%BE%E3%81%9F%E3%81%AF%E3%80%8C%E3%83%81%E3%82%A7%E3%83%83%E3%82%AF%E3%81%95%E3%82%8C%E3%81%A6%E3%81%84%E3%81%AA%E3%81%84%E3%80%8D)でチェックされている/チェックされていないを採用していたので、それに揃えることにしました :bow:
6f2540e